demo/src/app/app.routing.ts

```diff
@@ -24,6 +25,9 @@ export const routing = RouterModule.forRoot([
       { path: 'case/:jid/:ctid/:cid',
         component: CaseViewConsumerComponent,
         children: viewerRouting
+      },
```

Comment:
This is part of a future task in this epic: https://tools.hmcts.net/jira/browse/RDM-3918

Reply:
Seem to have highlighted wrong section, but I get it and have removed that bit.